landing/navbar.css

```diff
@@ -0,0 +1,146 @@
+.navbar-container-flex{
+    padding: 1rem 0.5rem;
+    justify-content: space-evenly;
+}
+
+
+.nav-title{
+    font-family: 'Norwester', sans-serif;                                         
+    font-size: 2rem;
+    display: flex;
+    gap: 5px;
+  
+}
+
+
+.flex-title{
+    display: flex;
+    flex-direction: column;
+}
+
+.nav-logo{
+    height: 2.5rem;
+    width: 2.5rem;
+}
+
+
+.nav-search-box{
+    border: 2px solid var(--lightgray-color);
+    padding: 0.5rem 2rem;
+}
+
+
+
+.search-bar{
+    width: 25rem;  
+    padding: 0 0.5rem;
+    font-size: 1rem;
+    outline: 2px transparent;
+    border: 1px solid white;
+}
+
+.search-btn-alone,.search-btn{
+    border: none;
+    color: var(--gray-color);
+    background-color: var(--white-color);
+    font-size: 1.2rem;
+}
+
+
+.navbar-icons{
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    gap:30px;
+}
+
+.navbar-icons:hover{
+    color: var(--baseteal-color);
+}
+
+.login-btn{
+    background-color: var(--baseteal-color);
+}
+
+.login-btn:hover{
+    background-color: var(--darkteal-color);
+}
+
+.search-btn-alone{
+    display: none;
+}
+
+@media only screen and (max-width: 1010px)
+{
+    .nav-search-box{
+        display: none;
+    }
+
+    .search-btn-alone{
+        display: block ;
+    }
+   
+}
+
+@media only screen and (max-width: 750px)
+{
+  
+    .search-btn-alone{
+        display: block;
+    }
+    .flex-title{
+        display: none;
+    }
+
+    
+}
+
+
+
+/* footer --------------------------------------------------------------------------- */
+div>[class|="child"]{
+    height: 10vh;
+    
+}
+
+/* .sapce-evenly{
+    display: flex;
+    justify-content: space-evenly;
+} */
+
+.footer-container-ec{
+    margin-left: auto;
```

Comment:
just a suggestion that you can use shorthand property of margin as margin : 0 auto ;

Reply:
ok